docs/node/guides/lab20.md

```diff
@@ -0,0 +1,98 @@
+---
+sidebar_position: 20
+---
+
+# Lab 20: Socket TCP
+
+This lab tutorial focuses on VILLASnode's [`socket`](../nodes/socket.md) node-type using a TCP connection. 
+
+An example is created to test the connection between VILLASnode's [TCP client (`socket`)](../nodes/socket.md) with a [Signal Generator (`signal`)](../nodes/signal.md) and VILLASnode's [TCP server (`socket`)](../nodes/socket.md). The server act as a echo server and is response for receiving the data from TCP client and send the same data back for testing purpose.
+
+## VILLASnode configuration file
+
+### TCP server
+
+``` url="external/node/etc/labs/lab20-server.conf" title="node/etc/labs/lab20-server.conf"
```

Comment:
This file is not present in the VILLASnode example configs:

https://github.com/VILLASframework/node/tree/master/etc/labs